cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -51,6 +51,14 @@ const std::string kWriteIOTime = "writeIOTime";
 // others
 const std::string kHiveDefaultPartition = "__HIVE_DEFAULT_PARTITION__";
 
+std::string getQueryId(const std::unordered_map<std::string, std::string>& confMap) {
+  auto it = confMap.find(kQueryTraceQueryId);
+  if (it != confMap.end()) {
+    return it->second;
+  }
+  return "";
+}
+
```

Comment:
A little bit confused with the empty query ID. Can we always give a meaningful ID to Velox?

Reply:
Can we uses the applicationId_stageId as the queryId?

Reply:
> Can we uses the applicationId_stageId as the queryId?

Do we have applicationId passed through JNI? Or could just align it with the [task name](https://github.com/apache/incubator-gluten/blob/b29aa3b9973d28ed03e9840772734dd1f47b3cdd/cpp/velox/compute/WholeStageResultIterator.cc#L87-L91) somehow.

Reply:
If `applicationId` is important for the feature and not yet passed through JNI, could also add it and use it for query context and task names / IDs in another PR. Thank you.

Reply:
The queryId concept is not really query id, it is velox query id, not Spark queryId, now one query for one task, so I think we can use the same name with task name.

Reply:
Now we don't pass the applicationId